src/reducers/selectorLocations.js

```diff
@@ -0,0 +1,26 @@
+import Immutable from 'immutable';
+
+import {SelectorLocations} from '../records';
+
+const emptyMap = new Immutable.Map();
+
+export default function reduceProjects(stateIn, action) {
```

Comment:
`reduceSelectorLocations`

Reply:
Also, on my list is to refactor all existing reducers to use `handleActions`, but for now I’d at least like to use it for new reducers (e.g. [`resizableFlex`](https://github.com/popcodeorg/popcode/blob/master/src/reducers/resizableFlex.js))

Reply:
Also generally we want to have unit tests for all reducers and sagas